docs/docs/tutorial/part-2/index.mdx

```diff
@@ -233,9 +235,31 @@ const AboutPage = () => {
 export default AboutPage
 ```
 
-2. In a web browser, visit `localhost:8000/about`. When your development server finishes rebuilding your site, the About page should look something like this:
+2. Add a page title: Gatsby lets you define a title and other [document head](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head) related tags with the [Gatsby Head API](/docs/reference/built-in-components/gatsby-head/).
+When you export a `Head` component from your page file, It automatically picks all the head tags in your JSX and adds them to the [document head](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head).
+This is important because it helps search engines like Google to better understand your site. For this tutorial we'll only be adding a title to this page but you can also add `<meta>`, `<links>` and other head related tags
+
+```js:title=src/pages/about.js
+import * as React from 'react'
+
+const AboutPage = () => {
+  return (
+    <main>
+      <h1>About Me</h1>
+      <p>Hi there! I'm the proud creator of this site, which I built with Gatsby.</p>
+    </main>
+  )
+}
+
+// Yay! We defined a page title. This shows up on the browser tab
+export const Head = () => <title>About Me<title/>
+
+export default AboutPage
+```
+
+3. In a web browser, visit `localhost:8000/about`. When your development server finishes rebuilding your site, the About page should look something like this:
```

Comment:
The "Key Gatsby Concept" below should also contain a mention of the Gatsby Head API

Reply:
Each of the key concepts down there (from here https://www.gatsbyjs.com/docs/tutorial/part-2/#use-the-link-component) seem to be talking about specific things like Link, Layout, and Stying, not sure where to slot that.

If we decide to create a new heading for the Head API then we'd have to elaborate and also stay consistent by adding a collapsible like  👇

<img width="785" alt="Screenshot 2022-08-16 at 07 13 51" src="https://user-images.githubusercontent.com/17142206/184810581-83857452-1cb4-41d2-a415-c173fd27f09a.png">